src/WindowsAzureMessaging/WindowsAzureMessaging/Helpers/MSInstallationHelper.h

```diff
@@ -0,0 +1,11 @@
+//----------------------------------------------------------------
```

Comment:
As far as I can tell, this file isn't used. Can we delete it?

Reply:
Removed